public/resources/stylesheets/page/_register.scss

```diff
@@ -83,7 +83,7 @@
 
 #nomessage {
     position: absolute;
-    bottom: -20px;
+    bottom: -5px;
     color: $font_w;
     font-size: 15px;
     font-weight: 800;
```

Comment:
using `.classes` instead of `#id` could make you work less preparing futures could use classes in more context